### PR TITLE
Roxanaluca assignment3

### PR DESCRIPTION
**Pull request by roxanaluca**

### CODE REVIEW

File: roxanaluca/assignment-3/Main/src/main/Main.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+                        
+                    }
+            }
+        }
+        return result;
+    }
+    static int[][] goTo =   {{1,1, 1, 0,-1,-1,-1,0},//for i
+                             {1,0,-1,-1,-1, 0, 1,1}};//for j
+
+    public static String wordSearch1(String[] set, String[] grid)
```
Comment (imorlowska) [inline roxanaluca/assignment-3/Main/src/main/Main.java]: Why the two implementations?

File: roxanaluca/assignment-3/Main/src/main/Main.java [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+import java.util.LinkedList;
+import java.util.Map;
+import java.util.Stack;
+
+public class Main {
+
+    /**
+     * @param args the command line arguments
+     */
+    public static void main(String[] args) {
```
Comment (imorlowska) [inline roxanaluca/assignment-3/Main/src/main/Main.java]: Now that you had the testing hangout, could you please add more tests?

File: roxanaluca/assignment-3/Main/src/main/Main.java (hunk 1 above)
Comment (roxanaluca) [reply roxanaluca/assignment-3/Main/src/main/Main.java]: One implementation is the solution to the problem and the other one is for the optional challenge.

**Comment by imorlowska** [comment]
Thanks for making these changes! Approved!

**Pull request merged**